docker-compose.yml

```diff
@@ -361,7 +361,7 @@ services:
       - LOG_LEVEL_SPRING_WEB=INFO
       - LOG_LEVEL_TOMCAT=DEBUG
       - LOG_LEVEL_DEFAULT=DEBUG
-      - LOG_LEVEL_LABKEY_DEFAULT=DEBUG
+      - LOG_LEVEL_LABKEY_DEFAULT=INFO
```

Comment:
Is this intentional? It's not mentioned in the PR description, so I wanted to double-check

Reply:
nope. Must have been leftover from testing. I'll revert. Thanks.